FSharp.Formatting.TestHelpers/TestHelpers.fs

```diff
@@ -23,16 +23,19 @@ let enableLogging() =
             source.Switch.Level <- System.Diagnostics.SourceLevels.All
             Logging.AddListener listener source)
 
+#if !NETSTANDARD2_0
+#if !NETCOREAPP2_1
```

Comment:
can we combine those, why exactly do we need this?

Reply:
Actually I can just remove them

Reply:
Done